timemachine/fe/energy_decomposition.py

```diff
@@ -0,0 +1,80 @@
+import functools
+from dataclasses import dataclass
+from typing import Callable, List, Sequence
+
+import numpy as np
+
+from timemachine.constants import BOLTZ, DEFAULT_TEMP
+
+Frames = np.ndarray
+Boxes = np.ndarray
+ReducedEnergies = np.ndarray
+Batch_u_fn = Callable[[Frames, Boxes], ReducedEnergies]
+
+
+@dataclass
+class EnergyDecomposedState:
+    """contains samples (frames, boxes) and a list of reduced energy functions"""
+
+    frames: np.ndarray
```

Comment:
Should this be `frames: Frames`?

Reply:
Updated in https://github.com/proteneer/timemachine/pull/936/commits/e1b9cf125b2f397f96f8d3388f7f1345778cfc55